app/js/home/list-service/tests/list.service.spec.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+describe('ListService (Service test)', function () {
+
+    var service;
+
+    beforeEach(angular.mock.module('main'));
+
+    beforeEach(angular.mock.module(function ($provide) {
+        $provide.constant('initialList', []);
+    }));
+
+    beforeEach(angular.mock.inject(function (_ListService_) {
+        service = _ListService_;
+    }));
+
```

Comment:
I would expect this test to only have one test method:
it('should contain initial todos')

Other tests can be unit tests.